--- origsrc/xrx-1.0.2/plugin/Makefile.am	2008-06-20 22:14:28.000000000 -0500
+++ src/xrx-1.0.2/plugin/Makefile.am	2008-10-27 00:44:23.037863900 -0500
@@ -24,7 +24,7 @@
 
 RXSRC = $(top_srcdir)/rx
 
-INCLUDES = -I$(RXSRC)
+INCLUDES = -I$(RXSRC) -I$(srcdir)/include
 
 # -DPLUGIN_TRACE can be added to the following definition to debug
 PLUGIN_DEFINES = -DXP_UNIX -DMOZ_X11 -DOJI
@@ -35,7 +35,7 @@
 AM_CFLAGS = -DXP_UNIX @XRX_CFLAGS@ @XRX_PLUGIN_CFLAGS@ @PLUGIN_CFLAGS@
 
 libxrx_la_LIBADD = @XRX_LIBS@ @XRX_PLUGIN_LIBS@
-libxrx_la_LDFLAGS = -avoid-version
+libxrx_la_LDFLAGS = -module -avoid-version -no-undefined
 
 PARAMS_SRCS = $(RXSRC)/PRead.c $(RXSRC)/PParse.c $(RXSRC)/BuildReq.c
 XMISC_SRCS = $(RXSRC)/XUrls.c $(RXSRC)/XAuth.c $(RXSRC)/XDpyName.c \
--- origsrc/xrx-1.0.2/xnest-plugin/Makefile.am	2008-06-20 22:25:57.000000000 -0500
+++ src/xrx-1.0.2/xnest-plugin/Makefile.am	2008-10-27 00:43:58.101161900 -0500
@@ -23,7 +23,7 @@
 RXSRC = $(top_srcdir)/rx
 PLUGINSRC = $(top_srcdir)/plugin
 
-INCLUDES = -I$(RXSRC)
+INCLUDES = -I$(RXSRC) -I$(PLUGINSRC)/include
 
 # -DPLUGIN_TRACE can be added to the following definition to debug
 PLUGIN_DEFINES = -DXP_UNIX -DMOZ_X11 -DOJI
@@ -49,4 +49,4 @@
 AM_CFLAGS = @XRX_CFLAGS@ @XRX_PLUGIN_CFLAGS@ @PLUGIN_CFLAGS@
 
 libxrxnest_la_LIBADD = @XRX_LIBS@ @XRX_PLUGIN_LIBS@
-libxrxnest_la_LDFLAGS = -avoid-version
+libxrxnest_la_LDFLAGS = -module -avoid-version -no-undefined
